Resources/Prototypes/Entities/Clothing/Belt/belts.yml

```
@@ -374,10 +374,10 @@
         whitelist:
           tags:
           - BotanyHatchet
-      # hydro:
-      #   whitelist:
-      #     tags:
-      #     - PlantAnalyzer # Dunno what to put here, should be aight.
+      hydro:
+        whitelist:
+          tags:
+          - PlantAnalyzer
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 12 but found 10

(indentation)